app/src/main/java/org/mozilla/fenix/onboarding/SyncCFRPresenter.kt

```diff
@@ -37,7 +37,7 @@ class SyncCFRPresenter(
     /**
      * Find the synced view and if it is available, then show the synced tab CFR.
      */
-    fun showSyncCFR() {
+    fun showSyncCFR(): Boolean {
```

Comment:
This change could be reverted if the suggestion to switch to a when statement is taken.